about.html

```diff
@@ -0,0 +1,119 @@
+            <div class="description">
+                <p>The Global Event brings together the community of professional, and activists who comprise the Spiritual World affiliate network in a different region every year. </p><p class="desc-desktop">This year, we hope to expand our invitation list including organizations and individuals. who want to participate and shared their experiences.</p><p class="desc-desktop">Lorem ipsum dolor sit amet consectetur adipisicing elit. Commodi libero at optio placeat qui quia veniam eius deleniti. Impedit vitae incidunt delectus aspernatur tempore pariatur expedita minus sunt natus quo.</p>
+            </div>
+            <p class="location">Please contact us per Email for any further questions about MV Fullstack Graduation Event 2023!</p>
+
+            <a class="hero-mail" href="gmail.com">swevent2023@swmyanmar.org <i class="fa-solid fa-chevron-right"></i></a>
+        </section>
+    </header>
+    <main>
+        <section class="event-logo">
+            <div class="container">
+                <div class="heading">
+                    <h2 class="heading-primary">MV Full-Stack Graduation 2023 Logo</h2>
+                    <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Earum, similique voluptatem.</p>
+                </div>
+                <div class="logo-section text-center">
+                    <img src="assets/images/icon.svg" alt="MV logo">
+                </div>
+            </div>
+        </section>
+
+        <section class="past-events">
+            <div class="container">
+                <div class="heading">
+                    <h2 class="heading-primary">See the past MV Full-Stack Graduations</h2>
+                    <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Earum, similique voluptatem.</p>
+                </div>
+                <div class="past-event text-center">
+                    <img class="img" src="assets/images/2021.png" alt="2021 event">
+                    <img class="img" src="assets/images/2022.png" alt="2022 event">
+                </div>
+            </div>
+        </section>
+    </main>
+
+    <footer>
+        <section class="partners">
+            <div class="container">
+                <div class="heading">
+                    <h2 class="heading-white">Partner</h2>
+                    <ul>
+                        <li class="img"><img src="assets/images/microverse-1.png" alt=""></li>
+                        <li class="img"><img src="assets/images/heineken.svg" alt=""></li>
+                        <li class="img"><img src="assets/images/google-2015.svg" alt=""></li>
+                        <li class="img"><img src="assets/images/airbnb.svg" alt=""></li>
+                        <li class="img"><img src="assets/images/mozilla-1.svg" alt=""></li>
+                    </ul>
+                </div>
+            </div>
+        </section>
+
+        <section class="footnote">
```

Comment:
- [ ] Good job so far But I would like to recommend you use a `div` element instead `section` tag for any cases where no heading is needed for semantic reasons. Please fix it for both index and about pages because it is a good practice   :+1: